lib/version-util.js

```diff
@@ -179,6 +179,16 @@ function findGreatestByLevel(versions, current, level) {
         .value();
 }
 
+/**
+ * @param versions  Array of all available versions
+ * @Returns         An array of versions with the release versions filtered out
+ */
```

Comment:
Great, this is a perfect helper function to reuse this functionality. However, I don't think it belongs in a separate file, since 1) it is only used locally within npm.js, and 2) the overhead of increasing the surface area of the version-util API doesn't justify the benefit in this case. It would be great to move it into npm.js directly. Hope that makes sense!